canine/localization/base.py

```diff
@@ -84,6 +84,8 @@ def __init__(
         scratch_disk_type: "standard" or "ssd". Default "standard".
         scratch_disk_size: size of scratch disk, in gigabytes. Default 10GB
         scratch_disk_name: name of scratch disk. Default is a random string
+        scratch_disk_job_avoid: whether to skip localization for job avoidence
+          if scratch disk already exists.
```

Comment:
I would change this to say "whether to bypass running script if scratch disk already exists. Default True; set to False if you want to retroactively modify contents of a scratch disk after it's been created and populated by another task."

Reply:
Added in e541a45